xds/third_party/protoc-gen-validate/import.sh

```diff
@@ -25,36 +25,26 @@ FILES=(
 validate/validate.proto
 )
 
-# clone the protoc-gen-validate github repo in /tmp directory
-pushd /tmp
-rm -rf $GIT_BASE_DIR
+# clone the protoc-gen-validate github repo in a tmp directory
+tmpdir="$(mktemp -d)"
+pushd "${tmpdir}" || exit 1
```

Comment:
Add `set -e` at the beginning of the script, then the whole script would exit if any line fails. Then you wouldn't need `|| exit 1`